tests/latex/LatexLintBearTest.py

```diff
@@ -17,3 +21,15 @@
 LatexLintBearTest = verify_local_bear(LatexLintBear,
```

Comment:
Why don't you put it inside LatexBearTest class? 😉 Then you don't need to import verify_local_bear anymore.